src/python/foglamp/core/scheduler.py

```diff
@@ -627,15 +632,24 @@ def max_running_tasks(self, value: int)->None:
         # the start of the awaited coroutine.
         args = self._process_scripts[schedule.process_name]
 
+        # add core management host and port to process script args
+        args_to_exec = args.copy()
+        args_to_exec.append("--port")
+        args_to_exec.append(str(self._core_management_port))
+        args_to_exec.append("--address")
+        args_to_exec.append("127.0.0.1")
```

Comment:
@MarkRiddoch - just want to make sure this goes through socket file... does it?

Reply:
Yes, this uses the loopback interface and does not go down to the network hardware - I am guessing this is the question you are asking.